dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -446,9 +450,10 @@ public Workflow refreshVersion(@ApiParam(hidden = true) @Parameter(hidden = true
      * @param user User who made call
      * @param workflowId ID of workflow
      * @param version Name of the workflow version
+     * @param hardRefresh refresh all versions, even if no changes
      * @return Updated workflow
      */
-    private Workflow refreshWorkflow(User user, Long workflowId, Optional<String> version) {
+    private Workflow refreshWorkflow(User user, Long workflowId, Optional<String> version, boolean hardRefresh) {
```

Comment:
Codacy found an issue: [The method 'refreshWorkflow(User, Long, Optional, boolean)' has an NPath complexity of 256](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6348842)